components/style/properties/properties.mako.rs

```diff
@@ -1141,7 +1141,10 @@ impl PropertyDeclaration {
                   matches!(*value, DeclaredValue::WithVariables(_))
               },
           % endfor
-          PropertyDeclaration::Custom(..) => true
+          PropertyDeclaration::Custom(_, ref value) => match *value {
```

Comment:
Is there any semantic difference between `CSSWideKeyword(_)` and `CSSWideKeyword(..)`?